lib/acfs/errors.rb

```diff
@@ -49,7 +49,8 @@ def initialize(opts = {})
       @stubs     = opts.delete :stubs
       @operation = opts.delete :operation
 
-      super opts, 'Ambiguous stubs.'
+      super opts, "Ambiguous stubs.\n"\
+        "#{stubs.map {|s| '  ' + s.opts.pretty_inspect }.join}"
```

Comment:
The string interpolation is not necessary; how about adding the space by joining:

```ruby 
stubs.map { |s| s.opts.pretty_inspect }.join('  ')
```